### DISCUSSION TITLE
1.20

### DISCUSSION THREAD

**Discussion by EysseW** [games647/FastLogin#1121] [General]
It doesn't seem to be working on 1.20.2 for me. I compiled latest commit. It DOES load up, but the mod has no effect. Running `/cracked` or `/premium` gives errors. What am I doing wrong? On java 17 jdk btw.

**Comment by TuxCoding** [comment]
> I compiled latest commit. 

You don't need to. The CI Jenkins server at codemc already builds the latest versions.

>Running /cracked or /premium gives errors.

You are likely missing a dependency. The best help we could give you if you send us this error message.
